packages/flutter_tools/lib/src/build_runner/build_script.dart

```diff
@@ -383,7 +390,9 @@ import "${path.url.basename(buildStep.inputId.path)}" as entrypoint;
 
 Future<void> main() async {
   registerPlugins(webPluginRegistry);
```

Comment:
doesn't matter for my use case but should this initialization line also be inside the if block?